### PR TITLE
add Binary general recog configure

### PR DESCRIPTION
**Pull request by Intsigstephon**

### CODE REVIEW

File: ppcls/arch/backbone/variant_models/__init__.py [hunk 1]
```diff
@@ -1,2 +1,3 @@
 from .resnet_variant import ResNet50_last_stage_stride1
 from .vgg_variant import VGG19Sigmoid
+from .lcnet_variant import PPLCNet_x2_5Tanh
```
Comment (cuicheng01) [inline ppcls/arch/backbone/variant_models/__init__.py]: PPLCNet_x2_5_Tanh，下同

File: ppcls/arch/backbone/variant_models/lcnet_variant.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import paddle
```
Comment (cuicheng01) [inline ppcls/arch/backbone/variant_models/lcnet_variant.py]: 文件名改成pp_lcnet_variant.py吧

File: ppcls/arch/backbone/variant_models/__init__.py (hunk 1 above)
Comment (Intsigstephon) [reply ppcls/arch/backbone/variant_models/__init__.py]: done.

File: ppcls/arch/backbone/variant_models/lcnet_variant.py (hunk 2 above)
Comment (Intsigstephon) [reply ppcls/arch/backbone/variant_models/lcnet_variant.py]: done

**Review by cuicheng01 — APPROVED** [review]

**Pull request merged**